### DISCUSSION TITLE
Happy holidays everyone!

### DISCUSSION THREAD

**Discussion by vlidholt** [serverpod/serverpod#507] [General]
![happy-holidays](https://user-images.githubusercontent.com/1539812/209399350-2f1bd910-3093-45c3-ba4d-12523f83af85.png)

Just wanted to send big thank you to everyone that have supported us throughout the year by giving us feedback, contributing code, and cheering us on! We cannot wait to share all the amazing things we have planned for 2023. 🥳

**Comment by yahu1031** [comment]
# Merry Christmas 🎊 🎅🏻 🦌 🎁 #
